homeassistant/components/volvooncall/__init__.py

```diff
@@ -74,9 +123,26 @@ async def async_setup_entry(
 
     volvo_data = VolvoData(hass, connection, entry)
 
-    hass.data[DOMAIN][entry.entry_id] = VolvoUpdateCoordinator(hass, volvo_data)
+    coordinator = hass.data[DOMAIN][entry.entry_id] = VolvoUpdateCoordinator(
+        hass, volvo_data
+    )
+
+    try:
+        await coordinator.async_config_entry_first_refresh()
+    except ConfigEntryAuthFailed:
+        return False
```

Comment:
In a follow up we should add a reauth flow for this case instead of returning False here.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#reauthentication

Reply:
Noted, and will do, thank you!